docs/sources/collect/datadog-traces-metrics.md

```diff
@@ -139,19 +139,19 @@ We recommend this approach for current Datadog users who want to try using {{< p
 
    Replace the following:
 
-    - _`<DATADOG_RECEIVER_HOST>`_: The hostname where the {{< param "PRODUCT_NAME" >}} receiver is found.
-    - _`<DATADOG_RECEIVER_PORT>`_: The port where the {{< param "PRODUCT_NAME" >}} receiver is exposed.
+   * _`<DATADOG_RECEIVER_HOST>`_: The hostname where the {{< param "PRODUCT_NAME" >}} receiver is found.
+   * _`<DATADOG_RECEIVER_PORT>`_: The port where the {{< param "PRODUCT_NAME" >}} receiver is exposed.
 
-Alternatively, you might want your Datadog Agent to send metrics only to {{< param "PRODUCT_NAME" >}}. 
+Alternatively, you might want your Datadog Agent to send metrics only to {{< param "PRODUCT_NAME" >}}.
 You can do this by setting up your Datadog Agent in the following way:
 
 1. Replace the DD_URL in the configuration YAML:
 
    ```yaml
     dd_url: http://<DATADOG_RECEIVER_HOST>:<DATADOG_RECEIVER_PORT>
    ```
-Or by setting an environment variable:
 
+Or by setting an environment variable:
```

Comment:
should this section be indented under the step number?

Reply:
I think so yes.

Reply:
Fixed in the last commit